tools/Organization-Dashboard/client/src/pages/activity.tsx

```
@@ -0,0 +1,271 @@
+  
+  const { data, isLoading, error } = useQuery<ActivityItem[]>({
+    queryKey: ['/api/activity']
+  });
+
+  // Filter activities based on search and activity type
+  const filteredActivities = data?.filter(activity => {
+    // Apply search filter
+    const searchMatches = 
+      searchQuery === "" || 
+      activity.description.toLowerCase().includes(searchQuery.toLowerCase()) ||
+      (activity.user?.name && activity.user.name.toLowerCase().includes(searchQuery.toLowerCase())) ||
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)